backend/pkg/modules/internal/bfla/bfladetector/learn_and_detect_bfla.go

```diff
@@ -254,82 +304,145 @@ func runtimeRecover() {
 	}
 }
 
+func (l *learnAndDetectBFLA) checkBFLAState(apiID uint, allowedStates []BFLAStateEnum) (BFLAState, recovery.PersistedValue, error) {
```

Comment:
```suggestion
func (l *learnAndDetectBFLA) checkBFLAState(apiID uint, allowedStates ...BFLAStateEnum) (BFLAState, recovery.PersistedValue, error) {
```

for a prettier syntax